examples/kubernetes/static_provisioning/static_provisioning_with_shared_cache.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: s3-pv
+spec:
+  capacity:
+    storage: 1200Gi # Ignored, required
+  accessModes:
+    - ReadWriteMany # Supported options: ReadWriteMany / ReadOnlyMany
+  storageClassName: "" # Required for static provisioning
+  claimRef: # To ensure no other PVCs can claim this PV
+    namespace: default # Namespace is required even though it's in "default" namespace.
+    name: s3-pvc # Name of your PVC
+  mountOptions:
+    - allow-delete
+    - region us-west-2
+    - cache-xz s3-csi-driver--usw2-az1--x-s3
```

Comment:
We should specify that this needs to be an Express One Zone bucket and also probably link to https://github.com/awslabs/mountpoint-s3/blob/main/doc/CONFIGURATION.md#shared-cache as there are a few suggestions there for permissions and having lifecycle rule etc.

Reply:
Done!